src/inimpkg/embedded.nim

```diff
@@ -1,2 +1,4 @@
 ## INim/src/embedded.nim: preloaded into user's REPL session
-## ---------------------------------------------------------
\ No newline at end of file
+## ---------------------------------------------------------
+{. warning[UnusedImport]:off .}
+import typetraits
```

Comment:
can you explain why this warning needs to be silenced?

Is it because `typetraits` is being imported for side-effects?

Reply:
Oh I see, it's pre-loaded. Clever!